packages/rebber/dist/escaper.js

```diff
@@ -22,9 +22,8 @@ var defaultEscapes = {
   '{': '\\{',
   '}': '\\}',
   '~': '\\textasciitilde{}'
-  /* Encode special characters in `value`. */
-
 };
+/* Encode special characters in `value`. */
```

Comment:
No changes here, please

Reply:
grr this is learna build that creates that.

Reply:
You can run it locally using `--scope=rebber-plugins`

Reply:
As I needed many plugins, I did not use scope but this is really annoying if the comman dresult is not consistant accross computers.